### PR TITLE
GH-91752: Skip test_freeze_simple_script when zlib is not available

### PR DESCRIPTION
**Pull request by AbhigyanBose**
gh-91752: Skip test_freeze_simple_script when zlib is not available

### CODE REVIEW

**Review by tiran — CHANGES_REQUESTED** [review]
Please move the blurb from ``Library`` to ``Tests`` category.

File: Lib/test/test_tools/test_freeze.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
 from test.support import os_helper
+from test.support import requires_zlib
 from test.test_tools import imports_under_tool, skip_if_missing
 
 skip_if_missing('freeze')
 with imports_under_tool('freeze', 'test'):
     import freeze as helper
 
-
+@requires_zlib()
```
Comment (tiran) [inline Lib/test/test_tools/test_freeze.py]:
```suggestion
@support.requires_zlib()
```

File: Lib/test/test_tools/test_freeze.py [hunk 2]
```diff
@@ -6,13 +6,14 @@
 
 from test import support
 from test.support import os_helper
+from test.support import requires_zlib
```
Comment (tiran) [inline Lib/test/test_tools/test_freeze.py]:
```suggestion
```

**Review by tiran — COMMENTED** [review]
You don't have to import the function, just it from support module directly.

**Review by tiran — APPROVED** [review]
LGTM, thanks for the quick fix!

**Comment by AbhigyanBose** [comment]
@tiran 

Is there anything else I need to do here ?
Or should I just wait for someone to merge this ?

**Review by gvanrossum — APPROVED** [review]
Looks like this simply fell between the cracks. Since @tiran approved it I am just going to merge it for you.

**Comment by gvanrossum** [comment]
Let's try 3.10, 3.9.

**Comment by gvanrossum** [comment]
I don't care enough about the backports, I'll remove the labels. @AbhigyanBose If you want to see if manually backporting makes sense, go ahead, but don't spend much time on it (the likely cause of the failure is that the code doesn't exist in that form in 3.10 or 3.9). Start with 3.10 first, if any.

**Comment by AbhigyanBose** [comment]
I'm testing if backport's needed right now. But this issue seems to occur only for fresh installs of the OS (some dependency for a different application resolved the missing module issue on my daily os, currently testing with Docker containers)

**Comment by AbhigyanBose** [comment]
Checked both 3.10 and 3.9, no backports required.

**Pull request merged**